modules/gltf/test/lib/extensions/EXT_feature_metadata.spec.js

```diff
@@ -12,7 +12,8 @@ test('gltf#EXT_feature_metadata - Should do nothing if no "EXT_feature_metadata"
     }
   };
 
-  await decodeExtensions(gltfWithFeatureTextures);
+  const options = {gltf: {loadImages: true, loadBuffers: true}};
```

Comment:
`{loadImages: true, loadBuffers: true}` those values are default . You don't have to set them explicitly.